src/commands/createHttpFunction.ts

```diff
@@ -85,7 +42,7 @@ async function getFunctionsApi(context: IActionContext): Promise<AzureFunctionsE
             await funcExtension.activate();
         }
 
-        return funcExtension.exports.getApi<AzureFunctionsExtensionApi>('^1.1.0');
+        return funcExtension.exports.getApi<AzureFunctionsExtensionApi>('^1.2.0');
```

Comment:
Just to be clear, you still have to update this api version in the functions extension

Reply:
Yeah, I'm going to fix that weird bug we saw in stand-up before I update the api version.